cvss.js

```diff
@@ -250,7 +250,16 @@ var CVSS = function (id, options) {
     l.appendChild(this.vector = e('a'));
     this.vector.className = 'vector';
     this.vector.innerHTML = 'CVSS:3.1/AV:_/AC:_/PR:_/UI:_/S:_/C:_/I:_/A:_';
-    
+    // setup the copy button/icon
+    l.appendChild(this.copyButton = e('button'))
+    this.copyButton.style.visibility = "hidden"
+    this.copyButton.className = "copyButton"
+    this.copyButton.innerHTML = '<img src="copy-icon.svg"/>'
+    this.copyButton.onclick = function () {
+        navigator.clipboard.writeText(document.querySelector(".vector").innerText)
```

Comment:
Did you check this on IE/Edge?
As per https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText
writeText doesn't work on IE Edge or IE. (because of this I had to do it differently on on Vulnogram https://vulnogram.github.io/#json Copy button).

Reply:
I've checked on Edge for Mac just now and it did work.  In order to use a `document.execCommand("copy")` technique we'll need some additional refactoring, since one can not call `.select()` on a `<a>` element unfortunately.  It seems that function is only available on text type elements.

MDN does also say `execCommand` is obsolete [here](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand). 

I don't mind doing some more fiddling though if you prefer using the `execCommand` approach, but I think it will require some rejiggering of the `.vector` element a bit.  I might be wrong, really not my expertise here!

What do you think?